armi/physics/fuelPerformance/plugin.py

```diff
@@ -59,46 +55,4 @@ def defineParameters():
     @staticmethod
     @plugins.HOOKIMPL
     def afterConstructionOfAssemblies(assemblies, cs):
```

Comment:
Should these three lines be deleted as well?

Reply:
They could be. I just put a `pass` into the body of the function just to acknowledge the presence of the hook without actually doing anything when its called.

Not sure what happens if you remove the hook entirely...

Reply:
I don't think this method is defined for all other plugins, so I think it can be removed without issue. @ntouran would probably know for certain

Reply:
I guess it is part of the base `plugin` class: https://terrapower.github.io/armi/.apidocs/armi.plugins.html?highlight=afterconstructionofassemblies

So I'll delete it here and let the fuel performance plugin just default to that of the parent class.